lang/system/pl.xml

```diff
@@ -275,30 +422,131 @@ Nie wyłączaj systemu ani nie zamykaj aplikacji.</warning_saving>
+			<select_file>Wybierz plik</select_file>
+			<select_directory>Wybierz katalog</select_directory>
+			<compatible_content>{} archiwum/wów znalezionych ze wspieraną zawartości.</compatible_content>
+			<successed_install_archive>Pomyślnie zainstalowano {} archiwum/wów ze wspieraną zawartością:</successed_install_archive>
+			<update_app>Aktualizuj aplikację do:</update_app>
+			<failed_install_archive>Instalacja zawartości z {} archiwum/wów nie powiodła się:</failed_install_archive>
+			<not_compatible_content>Kompatybilna zawartość nie jest dostępna w {} archiwum/wach:</not_compatible_content>
+			<delete_archive>Usunąć archiwum?</delete_archive>
+		</archive_install>
+		<license_install>
+			<successed_install_license>Licencja została zainstalowana pomyślnie.</successed_install_license>
+			<failed_install_license>Instalacja licencji zakończyła się n.
+Upewnij się, że pliki work.bin/rif lub klucz zRIF.</failed_install_license>
+		</license_install>
+		<reinstall>
+			<reinstall_content>Zainstalować ponownie tą zawartość?</reinstall_content>
+			<already_installed>Ta zawartość jest już zainstalowana.</already_installed>
+			<reinstall_overwrite>Czy chcesz ją zainstalować ponownie i nadpisać istniejące dane?</reinstall_overwrite>
+		</reinstall>
+	</install_dialog>
+
 	<live_area>
 		<start>Rozpocznij</start>
 		<continue>Kontynuuj</continue>
+		<help>
+			<control_setting>Using configuration set for keyboard in control setting</control_setting>
+			<firmware_not_detected>Oprogramowanie nie zostało wykryte. Instalacja jest zalecana.</firmware_not_detected>
+			<firmware_font_not_detected>Czcionki oprogramowania nie zostały wykryte. Instalacja jest zalecana dla czcionek tekstu w trybie 'Live Area'</firmware_font_not_detected>
+			<live_area_help>Pomoc 'Live Area'</live_area_help>
+			<browse_app>Browse in app list</browse_app>
+			<browse_app_control>D-pad, Left Stick, Wheel in Up/Down or using Slider</browse_app_control>
+			<start_app>Start App</start_app>
+			<start_app_control>Click on Start or Press on Cross</start_app_control>
+			<show_hide>Show/Hide Live Area during app run</show_hide>
+			<show_hide_control>Press on PS</show_hide_control>
+			<exit_livearea>Exit Live Area</exit_livearea>
+			<exit_livearea_control>Click on Esc or Press on Circle</exit_livearea_control>
+			<manual_help>Manual Help</manual_help>
+			<browse_page>Browse page</browse_page>
+			<browse_page_control>D-pad, Left Stick, Wheel in Up/Down or using Slider, Click on &lt;/&gt;</browse_page_control>
+			<hide_show>Ukryj/pokaż przycisk</hide_show>
+			<hide_show_control>Lewy przycisk myszy lub wciśnij Trójkąt</hide_show_control>
+			<exit_manual>Wyjdź z instrukcji</exit_manual>
+			<exit_manual_control>Naciśnij Esc lub wciśnij PS na kontrolerze</exit_manual_control>
+		</help>
 	</live_area>
+
+	<performance_overlay>
+		<avg>Średnia (Avg)</avg>
+		<min>Mimimum (Min)</min>
+		<max>Maksimum (Max)</max>
```

Comment:
The text exceeds the dialog

Reply:
Added polish shortcuts then, they are much the same just avg is different